content/development/handbook/git/_index.md

```diff
@@ -4,4 +4,18 @@ description: Information about version control with Git
 tags: ["development", "handbook", "git"]
 weight: 100
 ---
-TODO
\ No newline at end of file
+<div id="contributionContent"></div>
```

Comment:
Whats is this sneaky thing? :)

Reply:
I'm using a get req for the readme files in the altinn.studio repo.
That way we will have updated information on "how to get started" and "contribution" only by updating the readme files :)